<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverSettings</class>
 <widget class="QWidget" name="DriverSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Driver Full Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="driverFullName"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="driverCode"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="employerFullName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Employer Full Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Vehicle Code</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="vehicleCode"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Driver Phone</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Driver Code</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="driverPhone"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>driverFullName</tabstop>
  <tabstop>driverCode</tabstop>
  <tabstop>driverPhone</tabstop>
  <tabstop>employerFullName</tabstop>
  <tabstop>vehicleCode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
